kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -93,6 +93,18 @@ public class BigQuerySinkConfig extends AbstractConfig {
       "The interval, in seconds, in which to attempt to run GCS to BQ load jobs. Only relevant "
       + "if enableBatchLoad is configured.";
 
+  public static final String BQ_STREAMING_MAX_ROWS_PER_REQUEST_CONFIG =                     "bqStreamingMaxRowsPerRequest";
+  private static final ConfigDef.Type BQ_STREAMING_MAX_ROWS_PER_REQUEST_TYPE =              ConfigDef.Type.INT;
+  private static final Integer BQ_STREAMING_MAX_ROWS_PER_REQUEST_DEFAULT =                  50000;
+  private static final ConfigDef.Importance BQ_STREAMING_MAX_ROWS_PER_REQUEST_IMPORTANCE =  ConfigDef.Importance.LOW;
+  private static final String BQ_STREAMING_MAX_ROWS_PER_REQUEST_DOC =
```

Comment:
The maximum number of rows to be sent in one batch in the request payload to bigquery. 
This can reduce number of failed calls due to `Request Too Large` if the payload exceeds BigQuery specified quota limits. (https://cloud.google.com/bigquery/quotas#write-api-limits)
Setting it to a low value can result in degraded performance of the connector